tagging_core.py

```
@@ -745,8 +877,38 @@ def core_log_gui(message: str, color: str = "black") -> None:
+                # ルーチン結果なので GUI へは出さず debug log のみ（issue #10）。
+                n_unchanged += 1
+                log_dbg(_get_string_internal("TaggerCore", "Log_No_New_Tags", current_index_str=current_index_str, file_name=output_path.name))
+                continue
+            added_tags = tuple(merged[len(existing_tags):])
+            new_content = ", ".join(merged)
+        else:
+            # 新規作成（previous_content=None）または上書き
+            if output_path.is_file():
+                try:
+                    previous_content = output_path.read_text(encoding='utf-8')
+                except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)